chapter02/rwd/ex5-2.pl

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/perl
+
+print "Give me a string: ";
+my $s = <>;
+
+print "Give me a number: ";
+my $n = <>;
+
+print "The product is: " . $s x $n ."\n";
```

Comment:
This will print the first version of string on the same line as "The product is: " which I'm not sure is the expected behaviour (It says separate lines, separate from each other, separate from other printed stuff? Dunno)